### PR TITLE
Add driver scripts

### PR DESCRIPTION
**Pull request by yulong-plct**

### CODE REVIEW

File: intrinsic-test.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#传入工具链绝对路径，到riscv-gnu-toolchain即可
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:1]: 建议改成英文，以后应该会backport到rvv-intrinsic-doc

File: intrinsic-test.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+tool=
+#传入generator路径，具体到rvv-intrinsic-doc即可
+cases=
+
+while [ $# -gt 0 ]; do
+  case "$1" in
+    --gnu-prefix)
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:8]: how about `--gnu-toolchain-root`?

File: intrinsic-test.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      tool=$2
+      shift 2
+      ;;
+    --rvv-prefix)
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:12]: `--gnu-tests-root`? which indicates the root dir of the generated cases.

File: intrinsic-test.sh [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+      ;;
+    --)
+      shift
+      break
+      ;;
+  esac
+done
+shift "$(($OPTIND - 1))"
+
+#生成测试用例
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:24]: you can leave the test generation to the end-user.

File: intrinsic-test.sh [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+cd "$cases"/rvv-intrinsic-generator || exit
+make gen-test
+
+#执行测试用例
+cd "$tool" || exit
+mkdir gcc/gcc/testsuite/gcc.target/riscv/rvv/gnu-auto-generated/
+cp -r "$cases"/gnu-auto-generated/* "$tool"/gcc/gcc/testsuite/gcc.target/riscv/rvv/gnu-auto-generated/
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:31]: I suppose you need to copy the exp files ? For example, `cp gnu-auto-generated.exp gcc/gcc/testsuite/gcc.target/riscv/rvv/gnu-auto-generated/`.

File: intrinsic-test.sh [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+make report-gcc-newlib -j "$(nproc)"
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:32]: ./configure --prefix=${install_dir} \
  --with-arch=rv${_KIND}imafdcv \
  --with-abi=${abi} \
  --with-isa-spec=20191213 \
  --with-sim=qemu

make -j $(nproc) build-sim SIM=qemu
make report -j $(nproc) RUNTESTFLAGS="gnu-auto-generated.exp"

File: intrinsic-test.sh (hunk 3 above)
Comment (yulong-plct) [reply intrinsic-test.sh:12]: Good idea.

File: intrinsic-test.sh (hunk 4 above)
Comment (yulong-plct) [reply intrinsic-test.sh:24]: Does that mean I don't have to think about generating cases?

File: intrinsic-test.sh [hunk 7]
```diff
@@ -0,0 +1,42 @@
+#riscv-gnu-toolchain repository absolute root directory
+toolchain=
+#rvv-intrinsic-doc repository absolute root directory
+doc=
+#intrinsic cases driver file absolute directory
+driver=
+
+while [ $# -gt 0 ] ; do
+	case "$1" in
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:9]: format?

File: intrinsic-test.sh (hunk 4 above)
Comment (Incarnation-p-lee) [reply intrinsic-test.sh]: Yes.

File: intrinsic-test.sh [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+		--gnu-toolchain-root)
+			tool=$2
+			shift 2
+			;;
+		--rvv-doc-root)
+      doc=$2
+      shift 2
+      ;;
+    --driver-root)
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:18]: just `--gnu-tests-root`, which is more clear than driver.

File: intrinsic-test.sh [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+      ;;
+    --) shift;
+      break
+      ;;
+  esac
+done
+shift "$(($OPTIND -1))"
+
+#generate intrinsic cases
+cd $doc/rvv-intrinsic-generator
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:30]: better to leave it to the end-user. 
1. End-user run make gen-gnu-test
2. End-user provide the riscv-gnu-toolchain and generated test root to this script.

File: intrinsic-test.sh [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+make gen-test
+
+#make report
+cd $tool
+make clean
+./configure --prefix="$PWD/opt-rvv64gc" --with-arch=rv64gcv --with-abi=lp64d --with-isa-spec=20191213 --with-sim=qemu
+make -j $(nproc) build-sim SIM=qemu
+
+mkdir gcc/gcc/testsuite/gcc.target/riscv/rvv/gnu-auto-generated/
+cp $driver/gnu-auto-generated.exp gcc/gcc/testsuite/gcc.target/riscv/rvv/gnu-auto-generated/
```
Comment (Incarnation-p-lee) [inline intrinsic-test.sh:40]: you can simply assume `gnu-auto-generated.exp` is in the same folder of this script.

**Review by Incarnation-p-lee — APPROVED** [review]

**Comment by Incarnation-p-lee** [comment]
LGTM, thanks @yulong-plct !.

**Pull request merged**